float32 stretching_compliance
---
bool success
